packages/variable-editor/src/components/variables/data/validation-utils.ts

```diff
@@ -17,15 +17,17 @@ const validationMessageBelongsToRow = (row: Row<Variable>, validationMessage: Va
   return key === validationMessage.path;
 };
 
-export const containsError = (validationMessages: ValidationMessages) => {
-  return validationMessages.find(validationMessage => validationMessage.severity === 'ERROR');
-};
-
-export const containsWarning = (validationMessages: ValidationMessages) => {
-  return validationMessages.find(validationMessage => validationMessage.severity === 'WARNING');
+export const rowClass = (messages: Array<MessageData>) => {
```

Comment:
I'd keep the `rowClass` function in `ValidationRow` as that's the only place it's needed.